build/fix-api-docs.js

```diff
@@ -5,7 +5,26 @@ var apiPath = path.join(__dirname, '..', 'docs', 'api');
 var replacements = [
   {find: /\/docs\/guides\/(.+)\.md/g, replace: 'tutorial-$1.html'},
   {find: /tutorial-tech.html/g, replace: 'tutorial-tech_.html'},
-  {find: /\/docs\/guides\//g, replace: '#'}
+  {find: /\/docs\/guides\//g, replace: '#'},
+  {find: /(\<h[1-6] id="(?:.*)?)video-js(.*)?"\>/g, replace: '$1videojs$2">'},
+  {find: /(\<h[1-6] id="(?:.*)?)don-t(.*)?"\>/g, replace: '$1dont$2">'},
+  {find: /(\<h[1-6] id="(?:.*)?)node-js(.*)?"\>/g, replace: '$1nodejs$2">'},
+  {find: /(\<h[1-6] id="(?:.*)?)vtt-js(.*)?"\>/g, replace: '$1vttjs$2">'},
+  {find: /(\<h[1-6] id=")-(.*)("\>)/g, replace: '$1$2$3'},
+  {find: /(\<h[1-6] id=")(.*)-("\>)/g, replace: '$1$2$3'},
```

Comment:
headings with backticks get a dash in there. Need two rules to get those that start and end with them.